bigbluebutton-html5/imports/ui/components/layout/layout-manager/customLayout.jsx

```diff
@@ -168,6 +165,7 @@ const CustomLayout = (props) => {
                 },
               },
               cameraDock: {
+                position: CAMERADOCK_POSITION.CONTENT_TOP,
```

Comment:
Can we use the initial values as a fallback (in every layout) instead of always resetting them on layout change? 

That way, if the user adjusts the camera position in Custom layout, switches to another layout, and then returns to Custom, their changes will be preserved.

```suggestion
                position: cameraDock.position || CAMERADOCK_POSITION.CONTENT_TOP,
```


https://github.com/user-attachments/assets/13c680c7-7505-430a-afef-d07b969ffdb4